src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -786,7 +785,8 @@ define ReportMME:
 
 define MMEDateList: 
   (ReportMME) R
-  where R.End is not null and R.MMEValue is not null
+  where 
+    R.End is not null and R.MMEValue is not null and (year from Today() - year from R.CalculatedEnd <= 2)
```

Comment:
@mldorr Maggie, FYI, I have added here to limit the graph data to last two years.